test/node/src/safe-empty-where-in.test.ts

```diff
@@ -0,0 +1,916 @@
+import {
+  destroyTest,
+  initTest,
+  TestContext,
+  testSql,
+  expect,
+  createTableWithId,
+  DIALECTS,
+  NOT_SUPPORTED,
+} from './test-setup.js'
+
+import {
+  WithSafeArrayWhereInPlugin,
+  Generated,
+  Kysely,
+  DeleteResult,
+  UpdateResult,
+} from '../../../'
+
+for (const dialect of DIALECTS) {
+  describe(`${dialect}: safe empty array where in`, () => {
+    let ctx: TestContext
+    let db: Kysely<Database>
+
+    interface Person {
+      id: Generated<number>
+      firstName: string
+    }
+
+    interface Database {
+      safeEmptyArrayPerson: Person
+    }
+
+    beforeEach(async function () {
+      ctx = await initTest(this, dialect)
+
+      db = new Kysely<Database>({
+        ...ctx.config,
+        plugins: [new WithSafeArrayWhereInPlugin()],
+      })
+
+      await db.schema.dropTable('safeEmptyArrayPerson').ifExists().execute()
+      await createTableWithId(db.schema, dialect, 'safeEmptyArrayPerson')
+        .addColumn('firstName', 'varchar(255)')
+        .execute()
+
+      await db
+        .insertInto('safeEmptyArrayPerson')
+        .values([
+          {
+            firstName: 'John',
+          },
+          {
+            firstName: 'Mary',
+          },
+          {
+            firstName: 'Tom',
+          },
+        ])
+        .execute()
```

Comment:
why do we need a special table for this suite?

Reply:
whats the convention in this project? im doing this just so to avoid collisions with other tests suites if things are run in parallel